protocol.go

```diff
@@ -12,7 +12,7 @@ import (
 // be read from the wire, if Listener.ReaderHeaderTimeout is not set.
 // It's kept as a global variable so to make it easier to find and override,
 // e.g. go build -ldflags -X "github.com/pires/go-proxyproto.DefaultReadHeaderTimeout=1s"
-var DefaultReadHeaderTimeout = 200 * time.Millisecond
+var DefaultReadHeaderTimeout = 10 * time.Second
```

Comment:
Can you please add the reason here why this timeout? Or maybe just point to whatever const/var you mentioned is exposed by stdlib.

Reply:
There's no other reason than "it's a reasonable timeout".